### PR TITLE
change nav into dropdown using <details>

### PR DESCRIPTION
**Pull request by yapkahkeat0711**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,55 +30,61 @@ <h1>Simple.css Test Page</h1>
...
+				<li><a href="#text__details">Details / Summary</a></li>
+				<li><a href="#text__address">Address</a></li>
+				<li><a href="#text__hr">Horizontal rules</a></li>
+				<li><a href="#text__tables">Tabular data</a></li>
+				<li><a href="#text__code">Code</a></li>
+				<li><a href="#text__inline">Inline elements</a></li>
+				<li><a href="#text__comments">HTML Comments</a></li>
+			</ul>
+		</details>
+		<details>
```
Comment (lkhrs) [inline index.html:54]: Multiple dropdown menus cause undesirable one menu per line

File: simple.css [hunk 2]
```diff
@@ -611,4 +611,5 @@ progress::-moz-progress-bar {
 
 progress:indeterminate::-moz-progress-bar {
   background-color: var(--accent-bg);
-}
\ No newline at end of file
+}
```
Comment (lkhrs) [inline simple.css:614]: If no CSS changes have been made, better to discard all than submit empty lines.

Dropdown menus could use better styling in header (see: one menu per line issue, could use slightly better design to make it look like a menu)

**Review by lkhrs — CHANGES_REQUESTED** [review]
I appreciate you addressing my concerns in #99, however these changes need some work on the design and markup. I left comments in the review.

**Comment by kevquirk** [comment]
I have no idea what the point of this PR is, and you're not communicating. Closing this PR down.

**Comment by yapkahkeat0711** [comment]
i am trying to fix the problem stated and make the navigation bar become more reliable.

**Comment by kevquirk** [comment]
The nav bar is supposed to be simple, then you cam build upon it as you see fit.

**Comment by yapkahkeat0711** [comment]
Oh ok. I will try it.

**Pull request closed without merging**